source/overview_index.html.erb

```diff
@@ -32,4 +32,4 @@ weight: 2
   }
 </style>
 
-<%= javascript_include_tag("tablesort.js", defer: true, async: true) %>
+<%= javascript_include_tag("table_sort.js", defer: true, async: true) %>
```

Comment:
I changed the name of this script to make the asset from this project distinct from the `tablesort` directory in the  `node_modules` directory, which is also on the Sprockets import paths.